### ISSUE TITLE
Function not implement:'epoch_1.pth'->'/mnt/mmdetection/tools/workdir/fcos50_gn/latest.pth

### ISSUE THREAD

**Issue by xjl-le** [open-mmlab/mmdetection#2865]
Thanks for your error report and we appreciate it a lot.

**Checklist**
1. I have searched related issues but cannot get the expected help.
2. The bug has not been fixed in the latest version.

**Describe the bug**
![Firefox_Screenshot_2020-06-01T00-14-52 954Z](https://user-images.githubusercontent.com/55579267/83367799-eea8c700-a3e8-11ea-94aa-32f952646e35.png)

the error is : 
2020-06-0100：17：38， 960-mm det-INFO-Epoch[1] [9700/9773]Lr：0.01000， eta：1day， 1：07：15， time ：0. 835，data_time：0.037， memory ： 9336， Loss_cLs：0.
3487， Loss bbox：0.4650， Loss center ness：0.6090， Loss：1. 4226
2020-06-0100：18：20， 575-mm det-INFO-Epoch[1] [9750/9773]Lr：0.01000， eta：1day， 1：06：28， time ：0. 832，data_time：0.037， memory ： 9336， Loss_cLs：0.
3608， Loss bbox：0.4685， Loss center ness：0.6099， Loss：1. 4391
Traceback(most recent call last) ：
File”train.py”， Line 162， in<module>
main()
File”train.py”， Line 158， in main
meta=meta)
File”/mnt/new mm det/mm detection-master/mm det/apis/train.py”， Line 179， in train_detector
runner.run(data_Loaders， cfg.workfLow， cfg.total_epochs)
File”/root/mini cond a 3/envs/my cond a/Lib/python 3.7/site-packages/mmc v/runner/runner.py”， Line 384， in run
epoch_runner(data_Loaders[i] ， **kwargs)
File”/root/mini cond a 3/envs/my cond a/Lib/python 3.7/site-packages/mmc v/runner/runner.py”， Line 293， in train
_self.call hook('after train epoch')
File”/root/mini cond a 3/envs/my cond a/Lib/python 3.7/site-packages/mmc v/runner/runner.py”， Line 245， in calL_hook
getattr(hook， fn_name) (self)
File”/root/mini cond a 3/envs/my cond a/Lib/python 3.7/site-packages/mmc v/runner/dist_utils.py”， Line 74， in wrapper
return func(*args， **kwargs)
File”/root/mini cond a 3/envs/my cond a/Lib/python 3.7/site-packages/mmc v/runner/hooks/checkpoint.py”， Line 3l， in after_train_epoch
self.out_dir， save_optimizer=self.save_optimizer， **self.args)
File”/root/mini cond a 3/envs/my cond a/Lib/python 3.7/site-packages/mmc v/runner/runner.py”， Line 270， in save_checkpoint
mmc v.symLink(filename， osp.join(out_dir， 'latest.pth') )
File”/root/mini cond a 3/envs/my cond a/Lib/python 3.7/site-packages/mmc v/utils/path.py”， Line 38， in symLink
os.symLink(src， dst， **kwargs)
OSError：[Errno 38] Function not implemented：'epoch_l.pth'->'/mnt/new mm det/mm detection-master/tooLs/work/fco sr 50_gn/Latest.pth

[**Reproduction**]
1. What command or script did you run?
python train.py --config ../configs/fcos/fcos_r50_caffee_fpn_gn-head_4x4_1x_coco.py  --workdir ./workdir/fcos50_gn --gpu 2
```
A placeholder for the command.
```
2. Did you make any modifications on the code or config? Did you understand what you have modified?
I don't modify the code
3. What dataset did you use?
coco
**Environment**
cuda10,pytorch1.3 ,mmcv==0.5.8
1. Please run `python mmdet/utils/collect_env.py` to collect necessary environment infomation and paste it here.
sys.pLatform：Linux

Python：3.7.6(default，Jan82020，19：59：22)[GCC7.3.0]

CUDA available：True

CUDAHOME：/usr/LocaL/cuda

NVCC：Cuda compilation tools， release 10.0， V 10.0. 130

GPU 0， 1：GeForce RTX2080Ti

GCC：gcc(Ubuntu7.5.0-3ubuntul~18.04)7.5.0

Py To rch：1.3. 1

Py Torch compiling details：Py Torch built with：

-GCC 7. 3

-Intel(R) Math Kernel Library Version 2019.0.4 Product Build 20190411 for Intel(R) 64  architecture applications

InteL(R) MKL-DNN v 0.20.5(GitHash0125f28c61 clf 822 fd48570b4c1066f96fcb9b2e)

-OpenMP 201511(a.k.a.OpenMP 4.5)

-NN PACK is enabled

-CUDA Runtime 10. 1

-NVCC architecture fLags：-gen code； arch=compute_37， code=sm_37； -gen code； arch=compute_50， code=sm_50； -gen code； arch=compute_60， code=sm_60； -gen code； arch=comp

ute6l， code=sm 61； -gen code； arch=compute 70， code=sm 70； -gen code； arch=compute 75， code=sm_75； -gen code； arch=compute 37， code=compute 37 

-Cu DNN 7.6. 3

-Magma 2.5. 1

-Buildsettings：BLAS=MKL， BUILD_NAMED TENSOR=OFF， BUILD_TYPE=Release， CXX_FLAGS=-Wno-deprecated-fvisibility-in lines-hidden-f openmp-D USE_FB GEMM-D USE

_Q NN PACK-D USE_PY TORCH_Q NN PACK-02-fPIC-Wno-narrowing-wall-W extra-Wno-missing-field-initializers-Wno-type-Limits-Wno-array-bounds-Wno-unknown-prag m

as-wno-sign-compare-Wno-unused-parameter-Wno-unused-variable-Wno-unused-function-Wno-unused-result-Wno-strict-overfLow-Wno-strict-aliasing-Wno-erro

r=deprecated-decLarations-Wno-string op-overfLow-Wno-error=pedantic-Wno-error=redundant-decLs-Wno-error=old-style-cast-f diagnostics-color=always-f alig

ned-new-Wno-unused-but-set-variable-wno-maybe-uninitialized-fno-math-errno-fno-trapping-math-Wno-string op-overfLow， DISABLE NUMA=1， PERF WITH AVX=l， P

ERF_WITH_AVX 2=1， PERF_WITH_AVX 512=1， USE_CUDA=True， USE_EXCEPTION_PTR=1， USE_G FLAGS=0FF， USE_GLOG=OFF， USE_MKL=ON， USE_MKL DNN=ON， USE_MPI=OFF， USE_NC CL=ON，

USE NN PACK=ON， USE OPENMP=ON， USE STATIC DISPATCH=OFF，

Torch Vision ：0.4. 2

Open CV：4.1. 2

MMC V：0.5. 8

MMDetection：2.0.0+unknown

MM Detection Compiler：GCC 7. 5

MM Detection CUDA Compiler ：10. 0

2. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [e.g., pip, conda, source]
pip install ....
    - Other environment variables that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

**Error traceback**
If applicable, paste the error trackback here.
```
![Firefox_Screenshot_2020-06-01T00-14-52 954Z](https://user-images.githubusercontent.com/55579267/83368051-0cc2f700-a3ea-11ea-822a-1faf2978f1e3.png)

A placeholder for trackback.
```

**Bug fix**
If you have already iden
tified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!

**Comment by hellock** [comment]
`symlink()` is not implemented on some OS, you can add `create_symlink=False` in the checkpoint config.

**Comment by xjl-le** [comment]
thangks for you answer .but where is checkpoint config ?  looking  forward to your reply .thanks you

**Comment by hellock** [comment]
Add `checkpoint_config = dict(create_symlink=False)` to the config file.

**Comment by xjl-le** [comment]
thanks you for you help .

**Comment by xjl-le** [comment]
![Firefox_Screenshot_2020-06-01T09-26-56 021Z](https://user-images.githubusercontent.com/55579267/83395829-8defad80-a42d-11ea-99e2-0c629005f596.png)

Now ,I had modified the checkpoint_config,and it run successful ,but when it run next epoch ,it raises new error.how can i fix it .thanks lot

**Comment by xjl-le** [comment]
hi ,can you help solve this problem in the next days. I love the project you created. I hope it will get better in the future.thanks you

**Comment by xjl-le** [comment]
I know the cause of the error.after one epoch the procedure need assess result on val_test,but it only supports single gpu evaluation.so,I modeified evaluation(interval=12),so the problem is solved
